src/main/java/team/silvertown/masil/image/exception/ImageErrorCode.java

```diff
@@ -7,7 +7,8 @@
 @Getter
 @AllArgsConstructor
 public enum ImageErrorCode implements ErrorCode {
-    NOT_SUPPORTED_CONTENT(500_16000, "지원하지 않는 파일 형식 입니다");
+    NOT_SUPPORTED_CONTENT(500_16000, "지원하지 않는 파일 형식 입니다"),
+    FILE_IS_EMPTY(500_16000, "존재하지 않는 이미지 파일입니다.");
```

Comment:
파일 에러 코드가 겹치네요 `500_16001`로 가시죠!
에러 메시지도 `비어있는 파일입니다.`가 괜찮을 것 같고 마침표는 어떻게 통일할까요?

```suggestion
    FILE_IS_EMPTY(500_16001, "비어있는 파일입니다");
```